compose/sub-tiled/Containerfile

```diff
@@ -2,4 +2,4 @@ FROM ghcr.io/bluesky/databroker:latest
 
 RUN apt update
 RUN apt install git npm --yes
-RUN TILED_BUILD_PUBLIC_PATH=/tiled/ui/ pip install git+https://github.com/bluesky/tiled.git@main
+RUN TILED_BUILD_PUBLIC_PATH=/tiled/ui/ pip install tiled
```

Comment:
Moving to "Normal" pip, this might need a `--no-binary` flag to make sure we do not get the wheel.

The fact that we re-install from source is critical to make sure that the public path gets routed through the build system.

We should also make sure that there is a release of tiled from when the sub-path stuff got merged.

Reply:
Good call, added here. https://github.com/bluesky/bluesky-pods/pull/47/commits/5bf4ae60239a738b17f6da07535d6d2d3771223a

There are still fairly regular releases to Tiled: https://github.com/bluesky/tiled/compare/v0.1.0a116...main